web_src/js/components/RepoActionView.vue

```diff
@@ -286,51 +286,48 @@ export function initRepositoryActionView() {
 
 </script>
 
-<style scoped lang="less">
-
+<style scoped>
 .action-view-body {
   display: flex;
-  height: calc(100vh - 266px); // fine tune this value to make the main view has full height
+  height: calc(-166vh);
 }
 
-// ================
-// action view header
```

Comment:
Should we readd these comments?

Reply:
They should be there, excluding the `================` line.

Reply:
Restored them with the line.